libs/design/src/molecules/navbar/navbar.md

```diff
@@ -0,0 +1,21 @@
+# Navbar
+
+The navbar is a container to be used as a navigation bar. It will align the elements horizontally within the container.
+
+## Import 
+Import using 
```

Comment:
How would I import this into my module? Can we standardize "type" of documentation this across all components?